doc/ex/composite_layers.rb

```diff
@@ -4,7 +4,7 @@ module Magick
   class ImageList
     # Create a shadow image for each image in the list
     def shadow(x_offset = 4, y_offset = 4, sigma = 4.0, opacity = 1.0)
-      map { |frame| frame.shadow(x_offset, y_offset, sigma, opacity) }
+      collect { |frame| frame.shadow(x_offset, y_offset, sigma, opacity) }
```

Comment:
@Watson1978 what is the difference between `map` and `collect` in this case? Are they not aliased?

Reply:
Seems `collect` calls https://github.com/rmagick/rmagick/blob/0c09a7603f5ee3349ac5809ab36d9bac56b5ba8d/lib/rmagick_internal.rb#L1412-L1419 method and `map` calls `Enumerable#map` because ImageList includes `Enumerable` at https://github.com/rmagick/rmagick/blob/0c09a7603f5ee3349ac5809ab36d9bac56b5ba8d/lib/rmagick_internal.rb#L1243

There is no alias to `map`. :(

Reply:
I think it is better that we will introduce `map` alias to override `Enumerable#map` like `ImageList#collect` instead of this PR.

Reply:
@Watson1978 Agreed, I think they should behave the same. It's confusing if they don't.